presto-main/src/main/java/com/facebook/presto/server/security/SpnegoFilter.java

```diff
@@ -120,6 +122,16 @@ public SpnegoFilter(KerberosConfig config)
         }
     }
 
+    private static void verifyKerberosConfig(KerberosConfig config)
+    {
+        requireNonNull(config.getKerberosConfig(),
+                "Kerberos config file is not set");
```

Comment:
ditto on the error messages: something like "kerberos_config must be set when frontend Kerberos is enabled"